README.md

```diff
@@ -347,6 +347,20 @@ it will not be free. We think that this is an acceptable workaround for now, tho
 you may be better off using a Rails scheduling system and use a cron job or similar to spin up your enqueue
 for the actual, executable background task.
 
+# Testing
+
+It's possible to use Sqlite to emulate SQS when running the tests.
+
+To do so, change the value of the env SQS_QUEUE_URL as the following examples:
+
+```
+# to use a file
+SQS_QUEUE_URL=sqlite:/path/filename.sqlite3
+
+# to use the memory
+SQS_QUEUE_URL=sqlite:/memory
```

Comment:
To prevent confusion with an actual path on the filesystem, I would suggest using `sqlite:memory` or even `sqlite:` (without path specifier). Or, following more closely the SQLite API itself (but uglier), `sqlite::memory:`.

Sidenote: I'm used to [connection strings in SQLAlchemy](https://docs.sqlalchemy.org/en/13/core/engines.html#sqlite) which chooses to use the double slash (indicating _authority_) and then a database specifier which can be a path or empty. It doesn't look like this is a cross-platform standard though and before this PR we already settled on just following the colon directly with the path, so I won't suggest changing that.

Reply:
Indeed it's a good idea! 

I changed to your 1st suggestion: `sqlite3:memory` (but added the number 3 to be similar with the one we use for the file URI)